main/rfc1867.c

```diff
@@ -714,7 +713,7 @@ SAPI_API SAPI_POST_HANDLER_FUNC(rfc1867_post_handler) /* {{{ */
 		int content_type_len = (int)strlen(content_type_dup);
 		char *content_type_lcase = estrndup(content_type_dup, content_type_len);
 
-		php_strtolower(content_type_lcase, content_type_len);
+		zend_str_tolower(content_type_lcase, content_type_len);
```

Comment:
Good catch - I think this may have made sense regardless for http headers but outside of the scope of this RFC

e.g. this is done in https://golang.org/src/net/http/internal/ascii/print.go - the rfc from 1995 https://datatracker.ietf.org/doc/html/rfc1867 makes no mention of locale so I think it was assuming everything was using the english locale

Reply:
RFC 1867 says "Note that mime headers are generally required to consist only of 7-bit data in the US-ASCII character set."